tests/generic_tests.robot

```diff
@@ -0,0 +1,20 @@
+*** Settings ***
+Documentation       Test suite used with generic test cases
+Metadata            Version    0.0.1
+
+Library             SeleniumLibrary
+Resource            ../resources/keywords/deploy_application.resource
+
+Suite Setup         Set Library Search Order    SeleniumLibrary
+Suite Teardown      Tear Down The Test Suite
+Test Setup          Given The Browser Is On Openshift Home Screen
+Test Teardown       Close Browser
+
+
+*** Test Cases ***
+Scenario: Verify error message for duplicate provider account friendly name
```

Comment:
@ikanias Please update the test case name with Title Case as "Scenario: Verify Error Message for Duplicate Provider Account Friendly Name"

Reply:
Done